data-service/src/main/java/uk/gov/gchq/palisade/service/data/stream/config/AkkaRunnableGraph.java

```diff
@@ -43,7 +43,7 @@
 import java.util.concurrent.CompletionStage;
 
 /**
- * Configuration for the Akka Runnable Graph used by the data-service for the connection to the audit-service via a
+ * Configuration for the Akka Runnable Graph used by the Data Service for the connection to the Audit Service via a
```

Comment:
```suggestion
 * Configuration for the Akka Runnable Graph used by the Data Service to send messages to the Audit Service via a
```

Reply:
This comment doesn't appear to have been addressed.

Reply:
I'll echo this